Example/www/js/index.js

```diff
@@ -74,7 +74,8 @@ var app = {
     },
 
     login: function() {
-      var emailAddress = prompt("Type in an email address", "");
+      // Replace this email with your own
+      var emailAddress = "sample-email@test.com";
```

Comment:
We are running into an issue where prompts don't work on iOS 13 in our sample app. Currently hardcoding the email, you can change this to something more useful when you are testing using our sample app.